public/app/plugins/datasource/grafana-azure-monitor-datasource/dashboards/vMInsightsWorkspace.json

```diff
@@ -0,0 +1,915 @@
+            "resource": "$ws",
+            "resultFormat": "table",
+            "workspace": ""
+          },
+          "azureMonitor": {
+            "aggOptions": [],
+            "aggregation": "Average",
+            "allowedTimeGrainsMs": [60000, 300000, 900000, 1800000, 3600000, 21600000, 43200000, 86400000],
+            "dimensionFilter": "*",
+            "dimensionFilters": [],
+            "metricDefinition": "Microsoft.Compute/virtualMachines",
+            "metricName": "Percentage CPU",
+            "metricNamespace": "Microsoft.Compute/virtualMachines",
+            "resourceGroup": "$rg",
+            "resourceName": "$res",
+            "timeGrain": "auto",
+            "timeGrains": [],
+            "top": "10"
+          },
+          "azureResourceGraph": {
+            "resultFormat": "table"
+          },
+          "hide": false,
+          "insightsAnalytics": {
+            "query": "",
+            "resultFormat": "time_series"
+          },
+          "queryType": "Azure Log Analytics",
+          "refId": "A",
+          "subscription": "$sub",
+          "subscriptions": [
+            "4dc2cd39-7a89-43d8-bebe-8bb501359891",
```

Comment:
let's remove these

Reply:
Removed these and the ones below as well